batched/dense/impl/KokkosBatched_SVD_Serial_Impl.hpp

```diff
@@ -49,6 +49,32 @@ KOKKOS_INLINE_FUNCTION int SerialSVD::invoke(SVD_USV_Tag, const AViewType &A,
       sigma.data(), sigma.stride(0), work.data());
 }
 
+template <typename AViewType, typename UViewType, typename VViewType,
+          typename SViewType, typename WViewType>
+KOKKOS_INLINE_FUNCTION int SerialSVD::invoke(
+    SVD_USV_Tag, const AViewType &A, const UViewType &U, const SViewType &sigma,
+    const VViewType &Vt, const WViewType &work,
+    typename AViewType::const_value_type &tol) {
```

Comment:
This interface is fine, but could you make it so that there is only one version with ``tol`` defaulting to ``Kokkos::ArithTraits<value_type>::zero`` like you did in the internal layer?

Reply:
(As opposed to having two different overloads)